server/router/routes/workflow-history-handler.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2021 Uber Technologies Inc.
```

Comment:
moved `server/router.js` route handlers into separate files in `server/router/routes`